.github/workflows/no-dups.yml

```diff
@@ -14,4 +14,4 @@ jobs:
       run: sudo npm install -g jscpd
 
     - name: Check duplications
-      run: jscpd . --min-lines 3 --min-tokens 35 --threshold 0
+      run: jscpd . --min-lines 3 --min-tokens 60 --threshold 0
```

Comment:
In concise languages like python, duplication beyond 45 tokens results in hard-to-maintain code.
You needed to increase the token count here - was it because of duplication in the test-code?
Reconsider your test code - you already have documented the inputs and outputs in the test-data. Does the test code need to repeat the logic? Is the test code understandable by others? Will they find it easy to change? Or will other developers get confused on seeing repeated patterns, unsure of where to change?
Maybe we don't have all the answers, but it is important to keep asking :)